scripts/base/mallmadness.nut

```diff
@@ -0,0 +1,100 @@
+//
+
+weaponModelIDs <- [293, 259, 260, 261, 262, 263, 264, 265, 266, 267,
+                   268, 269, 270, 291, 271, 272, 273, 274, 275, 277,
+                   278, 279, 281, 282, 283, 284, 280, 276, 285, 286,
+                   287, 288, 289, 290];
```

Comment:
No, these should not be in a strict numerical order. Other scripts lazily defined modelID as `weaponID + 258` which resulted in the wrong pickup models.

Reply:
Leave this comment in the code, not just on GitHub